src/modules/frequency/infrastructure/frequency.dto.mapper.ts

```diff
@@ -8,6 +13,21 @@ type PrismaTeacherClass = {
   level: { name: string } | null;
   activity: { id: number; name: string } | null;
 };
+export type PrismaStudentGeneralFrequency = {
+  studentid:   number;
+  fullname:    string;
+  date:        Date;
+  generalattendanceallowed: "false" | "true";
+  status: FrequencyStatus;
```

Comment:
Why all lowercase here?
```suggestion
  studentId: number;
  fullName: string;
  date: Date;
  generalAttendanceAllowed: "false" | "true";
  status: FrequencyStatus;
```

Reply:
Porque o que é retornado pelo Prisma do banco de dados está em lowercase, apesar de, na `query`, eu utilizar o _alias_ em _camel case_.

Reply:
Entendi, obrigado